eventRouting/event_filter.go

```diff
@@ -0,0 +1,33 @@
+package eventRouting
+
+import (
+	"strings"
+
+	fevents "github.com/cloudfoundry-community/firehose-to-syslog/events"
+)
+
+//EventFilter Given an Event Filter out unwanted event
+type EventFilter func(*fevents.Event) bool
+
+//HasIgnoreField Filter out the event has ignored app filed
+func HasIgnoreField(event *fevents.Event) bool {
+	ignored, hasIgnoredField := event.Fields["cf_ignored_app"]
+	return ignored == true && hasIgnoredField
+}
+
+//NotInCertainOrgs Filter out events not in certain orgs
+func NotInCertainOrgs(orgFilters string) EventFilter {
```

Comment:
Same here, filter out logs where org match one of the exclude_orgs item.

Reply:
@cloutierf  excluding is not the purpose here and I think we should not be subjective on `should` without use cases
The purpose here to include orgs.... 
My client's intent and my here is to reduce the chatty logs and send each log to different downstream

Reply:
I reedit the comment of the commits for more clarification

Reply:
using comma instead of | in the new commit

Reply:
No issue there.